trident/utilities.py

```diff
@@ -450,16 +450,3 @@ def make_onezone_ray(density=1e-26, temperature=1000, metallicity=0.3,
     ray.domain_right_edge = ray.domain_right_edge.to('code_length')
 
     return ray
-
```

Comment:
It appears that the astropy error that led to this kluge has been fixed in the last three years.  I cannot find the PR that fixed it in astropy, but I can now import trident in its source directory and it doesn't fail out the way it used to.  So I'm pulling this.